plasma_framework/test/src/exits/fee/FeeClaimOutputToPaymentTxCondition.test.js

```diff
@@ -0,0 +1,222 @@
+const FeeClaimOutputToPaymentTxCondition = artifacts.require('FeeClaimOutputToPaymentTxCondition');
+
+const { constants, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { TX_TYPE, OUTPUT_TYPE } = require('../../../helpers/constants.js');
+const { hashTx } = require('../../../helpers/paymentEip712.js');
+const { buildUtxoPos, utxoPosToTxPos } = require('../../../helpers/positions.js');
+const { sign } = require('../../../helpers/sign.js');
+const {
+    PaymentTransactionOutput, PaymentTransaction, FeeTransaction, FeeClaimOutput,
+} = require('../../../helpers/transaction.js');
+
+contract('FeeClaimOutputToPaymentTxCondition', ([richFather, bob]) => {
+    const ETH = constants.ZERO_ADDRESS;
+    const DUMMY_BLOCK_NUN = 123;
+    const alicePrivateKey = '0x7151e5dab6f8e95b5436515b83f423c4df64fe4c6149f864daa209b26adb10ca';
+    let alice;
+
+    before('setup alice account with custom private key', async () => {
+        const password = 'password1234';
+        alice = await web3.eth.personal.importRawKey(alicePrivateKey, password);
+        alice = web3.utils.toChecksumAddress(alice);
+        web3.eth.personal.unlockAccount(alice, password, 3600);
+        web3.eth.sendTransaction({
+            to: alice,
+            from: richFather,
+            value: web3.utils.toWei('1', 'ether'),
+        });
+    });
+
+    beforeEach('setup contracts', async () => {
+        this.dummyFramework = constants.ZERO_ADDRESS;
+        this.condition = await FeeClaimOutputToPaymentTxCondition.new(
+            this.dummyFramework, TX_TYPE.FEE, OUTPUT_TYPE.FEE_CLAIM, TX_TYPE.PAYMENT,
+        );
+    });
+
+    describe('verify', () => {
+        const getTestData = () => {
+            const feeOutputs = [
+                new FeeClaimOutput(OUTPUT_TYPE.FEE_CLAIM, 1000, alice, ETH),
+            ];
+            const feeTx = new FeeTransaction(TX_TYPE.FEE, [buildUtxoPos(1000, 0, 0)], feeOutputs, DUMMY_BLOCK_NUN);
```

Comment:
The txData should be hash of blockNum  + token.
Not really important here I guess, but maybe better to be consistent...